### PR TITLE
DBZ-585 Renaming 'initial_schema_only' snapshot mode for SQL Server i…

### PR DESCRIPTION
**Pull request by gunnarmorling**
…nto 'schema_only'

https://issues.redhat.com/browse/DBZ-585

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, could you review and merge?

File: debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java [hunk 1]
```diff
@@ -338,6 +348,11 @@ public SqlServerConnectorConfig(Configuration config) {
 
         this.databaseName = config.getString(DATABASE_NAME);
         this.snapshotMode = SnapshotMode.parse(config.getString(SNAPSHOT_MODE), SNAPSHOT_MODE.defaultValueAsString());
+
+        if (snapshotMode == SnapshotMode.INITIAL_SCHEMA_ONLY) {
```
Comment (jpechane) [inline debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java]: @gunnarmorling I must admit I am not sure about the purpose of this code

**Comment by jpechane** [comment]
@gunnarmorling `/home/travis/build/debezium/debezium/debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java` is failing due to wrong fromatting

File: debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java]: LOL, that's a left-over. Sorry. It's removed now.

**Comment by gunnarmorling** [comment]
Updated to remove the obsolete code and apply formatting.

**Pull request merged**